Tests/TuistLoaderTests/Up/UpRomeTests.swift

```diff
@@ -0,0 +1,91 @@
+import Foundation
+import TSCBasic
+import TuistCore
+import TuistSupport
+import XCTest
+
+@testable import TuistLoader
+@testable import TuistSupportTesting
+
+final class UpRomeTests: TuistUnitTestCase {
+    var platforms: [Platform]!
+    var cachePrefix: String!
+    var upHomebrew: MockUp!
+    var rome: MockRome!
+    var subject: UpRome!
+
+    override func setUp() {
+        super.setUp()
+        platforms = [.iOS]
+        cachePrefix = "Swift_1_2"
+        rome = MockRome()
+        upHomebrew = MockUp()
+        subject = UpRome(platforms: platforms,
+                         cachePrefix: cachePrefix,
+                         upHomebrew: upHomebrew,
+                         rome: rome)
+    }
+
+    override func tearDown() {
+        platforms = nil
+        cachePrefix = nil
+        rome = nil
+        upHomebrew = nil
+        subject = nil
+        super.tearDown()
+    }
+
+    func test_init() throws {
+        let temporaryPath = try self.temporaryPath()
+        let json = JSON(["platforms": JSON.array([JSON.string("ios")]), "cachePrefix": "Swift_5_1"])
+        let got = try UpRome(dictionary: json, projectPath: temporaryPath)
+        XCTAssertEqual(got.platforms, [.iOS])
+        XCTAssertEqual(got.cachePrefix, "Swift_5_1")
+    }
+
+    func test_isMet_when_rome_is_not_met() throws {
```

Comment:
I'd recommend being consistent with how we structure the body of the tests in the rest of the codebase:

```swift
// Given
...

// When
...

// Then
...

```
Using those comments makes the tests easier to read and reason about.